radixdlt-java/src/main/java/com/radixdlt/client/atommodel/validators/RegisteredValidatorParticle.java

```diff
@@ -0,0 +1,68 @@
+/*

```

Comment:
I think this is the wrong license header for the library.  Should be the MIT header.

Reply:
Indeed, will replace.